src/exercise/02/02.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+
+def start():
+    print(
+        """
+    Welcome to Pokémon Finder!
+    --------------------------
+  """
+    )
+
+
+def get_choice():
+    return input("Which Pokémon do you want to know about...\n")
+
+
+def fetch_pokemon(choice):
+    req = requests.get("https://pokeapi.co/api/v2/pokemon/" + choice.lower())
+
+    if req.status_code == requests.codes.ok:
+        print(req.text)
+    else:
+        req.raise_for_status()
+
+
+# Run program
+
+start()
```

Comment:
Not a big deal, but it is a good practice to do this if you want this functions to be called only when you're running the file directly, like python 02.py:
```python
if __name__ == "__main__":
    # your function calls
```
This way, if you'd like to import fetch_pokemon in another file, it would not execute functions from line 28 to 30, because you're importing, not running it with python, I guess `if __name__ == "__main__":` would be the equivalent of a main() function in c

Reply:
Oh, so cool!

Reply:
done